Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/UsersController.cs

```diff
@@ -39,22 +36,37 @@ public async Task<IActionResult> GetEntry([FromRoute] int id)
             await _db.SaveChangesAsync();
             return Ok(entry);
         }
-        // POST api/values
+        [HttpPost("login")]
+        public ActionResult<Users> Login([FromBody] Users User)
+        {
+
+            var ExistingUser = _db.UsersEntries.Where(o => o.Email == User.Email).FirstOrDefault();
+            if (ExistingUser != null)
+            {
+                if (ExistingUser.Password != User.Password)
+                {
+                    return new JsonResult("{Status: \"Invalid Username or Password\"}");
+                }
+                return Ok(User);
+            }
+            else
+            {
+                return BadRequest();
+            }
+        }
```

Comment:
By consistency I mean there is no space line between line 56 and line 57. The space wont be called whitespace

Reply:
But at some places there is space